### DISCUSSION TITLE
Synchronize or resync without downloading

### DISCUSSION THREAD

**Discussion by fmarvega** [abraunegg/onedrive#1965] [Q&A] [answered]
I have modified the skip_dir parameter in the config file and I have created a sync_list file. However, for this changes to apply, if I have understood it correctly, a --resync is needed. When I start the --resync all the files and folders in my OneDrive are to be downloaded into my local folder OneDrive ignoring all the constraints in skip_dir and sync_list. The problem is that I do not have enough space in my laptop's hard drive to store all the files in my OneDrive. Is there any way to resync without downloading all the files?

I have succesully used the --synchronize --single directory option, but I would like to know if it is possible to make it more handy.

Thanks in advance!

**Comment by abraunegg** [comment]
@fmarvega 
> When I start the --resync all the files and folders in my OneDrive are to be downloaded into my local folder OneDrive ignoring all the constraints in skip_dir and sync_list.

This is *very* odd ....

Lets start with some basics:
* How did you install the application?
* What is your application version: `onedrive --version`
* What is your application configuration: `onedrive --display-config`
* What directory did you place your 'config' file and 'sync_list' in ?

**Reply by fmarvega** [reply]
Thanks for the answer.

- How did you install the application?
Following the instructions in the `INSTALL.md` file: install all dependencies for Fedora, activate DMD, clone repository, run configure and make, then install, and deactivate DMD.
- What is your application version?
v2.4.17-16-g69d4a64
- What is your application configuration?
```
onedrive version                             = v2.4.17-16-g69d4a64
Config path                                  = /home/fmarvega/.config/onedrive
Config file found in config path             = false
Config option 'sync_dir'                     = /home/fmarvega/OneDrive
Config option 'enable_logging'               = false
Config option 'log_dir'                      = /var/log/onedrive/
Config option 'disable_notifications'        = false
Config option 'min_notify_changes'           = 5
Config option 'skip_dir'                     = 
Config option 'skip_dir_strict_match'        = false
Config option 'skip_file'                    = ~*|.~*|*.tmp
Config option 'skip_dotfiles'                = false
Config option 'skip_symlinks'                = false
Config option 'monitor_interval'             = 300
Config option 'monitor_log_frequency'        = 5
Config option 'monitor_fullscan_frequency'   = 12
Config option 'dry_run'                      = false
Config option 'upload_only'                  = false
Config option 'download_only'                = false
Config option 'local_first'                  = false
Config option 'check_nosync'                 = false
Config option 'check_nomount'                = false
Config option 'resync'                       = false
Config option 'resync_auth'                  = false
Config option 'classify_as_big_delete'       = 1000
Config option 'disable_upload_validation'    = false
Config option 'bypass_data_preservation'     = false
Config option 'no_remote_delete'             = false
Config option 'remove_source_files'          = false
Config option 'sync_dir_permissions'         = 700
Config option 'sync_file_permissions'        = 600
Config option 'application_id'               = 
Config option 'azure_ad_endpoint'            = 
Config option 'azure_tenant_id'              = common
Config option 'user_agent'                   = 
Config option 'force_http_2'                 = false
Config option 'debug_https'                  = false
Config option 'rate_limit'                   = 0
Config option 'operation_timeout'            = 3600
Config option 'sync_root_files'              = false
Selective sync 'sync_list' configured        = false
Config option 'sync_business_shared_folders' = false
Business Shared Folders configured           = false
Config option 'webhook_enabled'              = false
```
- What directory did you place your 'config' file and 'sync_list' in ?
'/home/fmarvega/onedrive'

**Comment by abraunegg** [comment, marked as answer]
@fmarvega 
Thanks for your responses.

Please can you re-read the [usage](https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md) document in regards to where configuration files and the 'sync_list' file is meant to be placed.

When they are in the *correct* location, when you use `--display-config`, the following will flip to 'true':
```
Config file found in config path             = true
```
and the output will display all of your 'skip_dir' and 'sync_list' configuration.

You can then test your configuration by adding `--dry-run` to your command line to validate if this is working as per your expectation.

**Reply by fmarvega** [reply]
Thanks for your help!
